<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>631</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="toolTip">
       <string>Video/Playlist URL</string>
      </property>
      <property name="text">
       <string>YouTube URL</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="filename_override"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_load_url">
      <property name="toolTip">
       <string>Load stream data for the provided URL.

Large playlists can take time to fully populate!</string>
      </property>
      <property name="text">
       <string>Load URL</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="url">
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="output_path">
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="toolTip">
       <string>Directory to store the downloaded files</string>
      </property>
      <property name="text">
       <string>Output Directory</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn_browse">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="toolTip">
       <string>Overrides the Video Title for the filename.
Does not affect the file extension.
If multiple streams are downloaded at once, the encoding will be added to the filename as well.

If downloading from a playlist (more than one video loads in the stream list)
then the override will prepend the video title.</string>
      </property>
      <property name="text">
       <string>Filename Override</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="toolTip">
       <string>HTTP/HTTPS proxy.
Can provide both HTTP and HTTPS if you wish, comma separated.
Most commonly used is HTTPS.

NOTE: You need HTTP proxy for HTTP URLs, and HTTPS proxy for HTTPS URLs!</string>
      </property>
      <property name="text">
       <string>Proxy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="proxies"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>350</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QProgressBar" name="progress_status">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn_download">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Select Streams to Download</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QProgressBar" name="progress_total">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="status_text">
      <property name="text">
       <string>Stream Progress</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="status_total">
      <property name="text">
       <string>Total Progress</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="thumbnail_preview">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>331</width>
     <height>269</height>
    </rect>
   </property>
   <property name="text">
    <string>Thumbnail Preview</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="stream_tree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>631</width>
     <height>341</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
